packages/user-operation-controller/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/user-operation-controller",
-  "version": "17.0.0",
+  "version": "17.0.1",
```

Comment:
In reference to above, should we bump this to 18.0.0 since we're bumping the `keyring-controller` peer dependency?

``` suggestion
  "version": "18.0.0",
```

Reply:
Why would this one be a major release?

Reply:
Peer dependencies are different than regular dependencies because they place a stipulation on how the consumer needs to change their dependencies. In this case, if a client wants to use `user-operation-controller` it also needs to instantiate `keyring-controller`, since `user-operation-controller` talks to it. That means both controller packages need to be dependencies of the client. This also means that the client needs to always use a version of `keyring-controller` that is compatible with `user-operation-controller`. So if `user-operation-controller`'s peer dependency on `keyring-controller` is bumped, it needs to be released under a major version bump. This way, if the client upgrades `user-operation-controller` it is forced to also upgrade `keyring-controller` to maintain compatibility.